packages/pdfx/android/src/main/kotlin/io/scer/pdfx/Messages.kt

```diff
@@ -207,7 +207,7 @@ class Messages(private val binding : FlutterPlugin.FlutterPluginBinding,
     }
 
     override fun registerTexture(): Pigeon.RegisterTextureReply {
-        val surfaceTexture = binding.textureRegistry.createSurfaceTexture()

```

Comment:
`Surface` produced with `SurfacerProducer` could change, and client is responsible on handling this changes
see https://docs.flutter.dev/release/breaking-changes/android-surface-plugins

Reply:
issue confirmed, fixing in https://github.com/ScerIO/packages.flutter/issues/563